apps/speed.c

```diff
@@ -1040,33 +1038,31 @@ static int ECDSA_verify_loop(void *args)
 /* ******************************************************************** */
 static long ecdh_c[EC_NUM][1];
 
-static int ECDH_compute_key_loop(void *args)
+static void ECDH_EVP_derive_key(unsigned char *derived_secret,
```

Comment:
Does writing this wrapper function buy anything over and above making the call directly at line 1061? At the moment it doesn't seem to add anything?